src/annotation_tool/project.py

```diff
@@ -129,27 +131,20 @@ def set_tsv_file(self, to : str):
         self.tsv_file = to
         if not os.path.exists(self.tsv_file):
             return
-        with open(self.tsv_file) as file:
-            first = True
-            for line in file:
-                if first:
-                    first = False
-                    continue
+        with open(to, newline='') as file:
+            reader = csv.DictReader(file, delimiter='\t')
+            for row in reader:
                 annotation = Annotation()
-                segments = line.replace("\n", "").split("\t")
-                for segmentNum in range(len(self.TSV_HEADER_MEMBERS)):
-                    value = segments[segmentNum]
-                    if getattr(annotation,
-                            self.TSV_HEADER_MEMBERS[segmentNum]) is int:
+                for header in self.TSV_HEADER_MEMBERS:
+                    value = row[header]
+                    if getattr(annotation, header) is int:
```

Comment:
It might also be worth to implement this all into the annotation constructor. So one would only call `Annotation(row)` maybe also together with `csv.DictReader`